### ISSUE TITLE
Use actual ead manifest file (contains stub of solution)

### ISSUE THREAD

**Issue by Balnian** [BellezaEmporium/galaxy-integration-ead#12]
**Describe the bug**
Use the actual ead local manifest for games owned and installed

**To Reproduce**
In 2023 I followed [this article](https://erri120.github.io/posts/2023-01-18/) on how to read and unencrypt ead manifest's and this is the python script I wrote (see attachments).
But I was never able to properly package it into a GOG Galaxy 2 plugin, so I hope this will be useful to you or some other contributor.

Some other remarks from what I remember more than a year ago when I last touched this:
* I had trouble packaging because `Cryptodome` is a weird package and couldn't be found/packaged correctly for a GOG plugin
   * I had started a rewrite with another cripto lib but never got it working (seriously why encrypt a launcher manifest file?)
* I think the python runtime used by Galaxy doesn't allow you to access the required WMI info (sandbox?) (need to be checked again)

**Expected behavior**
Get revenge on EA for breaking something that used to work (original plugin)

**Attached logs**
[ead.py.txt](https://github.com/user-attachments/files/17719695/ead.py.txt)
[requirements.txt](https://github.com/user-attachments/files/17719696/requirements.txt)

**Comment by Nutzzz** [comment]
Decrypting the IS file was the first method used in this fork (see issue upstream#44 and PR #2 here) and you'll see discussions as we worked through various issues, including those you bring up. The end result mostly worked, but was quite hacky.

The Juno API method (see branches) from which the releases were made works much better.

**Comment by BellezaEmporium** [comment]
EN : Thank you for your proposal. As @Nutzzz said, it was our first bet at the beginning of the rework, thinking it should be the easiest way of getting the data we wanted. 

Though, it added a layer of complexity, because accessing the IS file needed us administrative access to it (the file was in a protected area). Just opening it made Python leverage a UAC check at each launch when there was a byte of difference.

We had managed to load it up (and no, no worries about sandboxes or anything, the data was correct !), but it was too cumbersome to keep it afterwards, hence why I've decided to go for a less painful approach, let's say ? By actually querying the GraphQL API for the necessary data, and doing the rest on board.

FR : Merci pour votre proposition. Comme @Nutzzz l'a indiqué, il s'agissait de notre première idée au début de la reconstruction de ce plugin, en pensant que ce serait le meilleur moyen d'accéder aux données souhaitées.

Cependant, cela à ajouté une couche de complexité, car accéder au fichier IS nous demandait des droits administrateurs (le fichier étant dans un chemin protégé). Le simple fait de l'ouvrir faisait que Python envoyait une demande d'accès en mode administrateur dès qu'il y avait un bit de différence.

Nous avions réussi à l'ouvrir (et non, aucun souci de "bac à sable" ou quelconque problématique de ce genre, les données étaient correctes !), mais il était assez encombrant à garder au fil du temps, et c'est pourquoi j'ai décidé d'opter pour une approche moins douloureuse, on peut dire ? En appelant directement l'API GraphQL pour les données qui nous étaient nécessaires, et faire le reste directement.
